tests/5.0/target_update/test_target_update_mapper_to_discontiguous.c

```diff
@@ -32,26 +32,35 @@ int target_update_to_mapper() {
   OMPVV_TEST_OFFLOADING;
 
   newvec_t s;
+  int errors;
 
   s.data = (double *)calloc(N,sizeof(double));
   s.len = N;
- 
-  //initalize the struct
-  for (i = 0; i < s.len; i++) {
-    s.data[i] = i;
-  }
+  #pragma target data map(tofrom: s)
+  { 
+    //update array in host values
+    for (i = 0; i < s.len; i++) {
+      s.data[i] = i;
+    }
+    // update even array position values from host
+    // This should set them to i  
+    #pragma omp target update to(s.data[0:s.len:2])
 
-#pragma omp target update to(s)
+    //update array on the device
+    #pragma omp target //map(alloc: result[0:N])
+    for (i = 0; i < s.len; i++) {
+      s.data[i] = i;
```

Comment:
This needs to be `s.data[i] += i` and not an assignment.

Reply:
Fixed, thanks